content/docs/manage/organizations.md

```diff
@@ -1,451 +1,60 @@
-    }
-    // ... More projects
-  ]
-}
-```
-
-</details>
-
-#### Project-level metrics (for the current billing period)
-
-To get basic billing period-based consumption metrics for each project in the organization `org-ocean-art-12345678`, include `org_id` in the `GET /projects` request for consumption metrics:
-
-```bash shouldWrap
-curl --request GET \
-     --url 'https://console.neon.tech/api/v2/projects?org_id=org-ocean-art-12345678' \
-     --header 'accept: application/json' \
-     --header 'authorization: Bearer $NEON_API_KEY'
-```
-
-See more details about using this endpoint on the [Manage billing with consumption limits](/docs/guides/partner-billing#retrieving-metrics-for-all-projects) page in our Partner Guide.
-
-### List all organizations you belong to
-
-You can use the `GET /users/me/organizations` request to retrieve a list of all organizations associated with your personal account.
-
-```bash shouldWrap
-curl --request GET \
-     --url 'https://console.neon.tech/api/v2/users/me/organizations' \
-     --header 'accept: application/json' \
-     --header 'authorization: Bearer $NEON_API_KEY'
-```
-
-The response will include details about each organization, including the `org_id`, name, and creation date.
-
-### Example Response
-
-```json
-{
-  "organizations": [
-    {
-      "id": "org-morning-bread-81040908",
-      "name": "Morning Bread Organization",
-      "created_at": "2022-11-23T17:42:25Z",
-      "updated_at": "2022-12-04T02:39:25Z"
-    },
-    ...
-  ]
-}
-```
-
-## Managing organizations using the Neon CLI
-
-During private preview, we'll be continuing to add options for managing your organization using the Neon CLI. For now, we've added the main command `neonctl orgs` with a single subcommand `list`, which outputs a list of all organizations that the CLI user currently belongs to.
-
-Example:
-
-```bash
-neon orgs list
-Organizations
-┌────────────────────────┬──────────────────┐
-│ Id                     │ Name             │
-├────────────────────────┼──────────────────┤
-│ org-ocean-art-12345678 │ Example Org      │
-└────────────────────────┴──────────────────┘
-```
-
-For more detail, see [Neon CLI commands - orgs](/docs/reference/cli-orgs). We'll update this page as new management options become available.
-
-## Feature limitations
-
-As we continue to refine our Organizations feature during this private preview phase, please remember that these features are only available under paid account plans. Here are some temporary limitations you should be aware of:
+As we continue to refine our Organizations feature during this Early Access phase, please remember that these features are only available under paid account plans. Here are some temporary limitations you should be aware of:
 
 - **Integration limitations** — You cannot install new Vercel integrations on organization-owned projects. However, existing integrations will continue to work on projects transferred from personal to the organization account.
```

Comment:
Should we add these limitations to the Vercel and GitHub integration pages as well? Might deflect some issues.

Reply:
Yes, we can do that in a follow-up issue where we mention orgs in various places outside of the orgs content.